### PR TITLE
[RAPTOR-6038] Removing custom tasks docs in DRUM repo

### PR DESCRIPTION
**Pull request by drenfr01**
…docs for tasks

# This repository is public. Do not put here any private DataRobot or customer's data: code, datasets, model artifacts, .etc.

## Summary

Removed section in Readme.md and quickstart guide that are now in the official docs.

## Rationale

We don't want to keep two, duplicate sources of documentation. The docs in this repo are currently out of date, incomplete, etc. so we're iteratively moving everything to the official DataRobot docs

### CODE REVIEW

**Review by 4dahalibut — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -21,7 +21,7 @@ debugging, testing, and running your training and inference models with DataRobo
 This repository addresses the DataRobot functionality known as `custom models`. The terms `custom model` and `user model` can be used interchangeably, as can `custom model directory` and `code directory`.
 
 ## Quickstart <a name="quickstart"></a>
-The following example shows how to use the [DRUM](https://github.com/datarobot/datarobot-user-models/tree/master/custom_model_runner) tool to make predictions on an [sklearn regression model](model_templates/python3_sklearn). For the training model quickstart, please reference [this document](QUICKSTART-FOR-TASKS.md)
+The following example shows how to use the [DRUM](https://github.com/datarobot/datarobot-user-models/tree/master/custom_model_runner) tool to make predictions on an [sklearn regression model](model_templates/python3_sklearn).
```
Comment (eric-s-s) [inline README.md:24]: two things:
- while you're here, can you fix the line spacing to something close to 100 characters, please.
- and when i test your new link in pycharm it doesn't redirect.  have you tested it with a doc renderer?

**Comment by eric-s-s** [comment]
if the documentation is at docs.datarobot.com , can we have a link in the readme pointing to that?

**Review by eric-s-s — APPROVED** [review]
two nits but otherwise good.

1. confirm the new link.
2. maybe have a pointer to the new docs website

**Comment by drenfr01** [comment]
@eric-s-s made your new change! mind approving after the tests pass?

File: README.md [hunk 2]
```diff
@@ -17,11 +17,18 @@
 The **DataRobot User Models** repository contains information and tools for assembling,
 debugging, testing, and running your training and inference models with DataRobot.
 
+For further documentation on this and all other features please visit our comprehensive
+documentation at: https://docs.datarobot.com/
```
Comment (eric-s-s) [inline README.md:21]: perhaps link to: https://docs.datarobot.com/en/mlops/deployment/custom-models/index.html

and it looks like there's nothing in there for custom tasks, correct?

**Comment by eric-s-s** [comment]
i cannot find this in official docs.  where should i be looking?

**Review by eric-s-s — APPROVED** [review]

**Comment by drenfr01** [comment]
jarvis test cmrunner_integration

**Pull request merged**